server/src/main/java/com/objectcomputing/checkins/services/skills/Skill.java

```diff
@@ -14,9 +14,21 @@
 import java.util.UUID;
 
 @Entity
-@Table(name ="skills")
+@Table(name = "skills")
 public class Skill {
 
+    @Id
+    @Column(name = "skillid")
+    @AutoPopulated
+    @TypeDef(type = DataType.STRING)
+    private UUID skillid;
+    @NotBlank
+    @NotNull
+    @Column(name = "name", unique = true)
+    private String name;
+    @Column(name = "pending")
```

Comment:
required= true

Reply:
pending isn't a required field. If the user leaves it out it is set to true.